go-controller/pkg/metrics/ovn_db.go

```diff
@@ -318,20 +318,41 @@ var (
 	sbDbSchemaVersion string
 )
 
+func getDbSockPath() (string, error) {
```

Comment:
@npinaeva should we name this `getNBDBSockPath()` since it's NB specific? Or maybe it should instead take a "direction" parameter that it can sprintf into ovn*_db.sock?

Reply:
Let's just rename to `getNBDBSockPath()` because I don't see any other usages for this function for now